lib/src/bloc/movies_bloc.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+
+import '../model/movies_model.dart';
+import '../resources/movies_repository.dart';
+import 'IMoviesFetcherBloc.dart';
+
+class MoviesBloc extends IMoviesFetcherBloc {
+  MoviesRepository moviesRepository = MoviesRepository();
+  StreamController<MoviesModel> moviesStreamController = StreamController();
```

Comment:
Both variables are private inside bloc, add '_' at beginning

Reply:
I have the doubt if that was necessary, great, thanks, Added